example/spec/emsReport/emsReportSpec.js

```diff
@@ -0,0 +1,71 @@
+const moment = require('moment');
+const { Lambda } = require('aws-sdk');
+const { loadConfig } = require('../helpers/testUtils');
+const aws = require('@cumulus/common/aws');
+
+const config = loadConfig();
+
+const emsReportLambda = `${config.stackName}-EmsReport`;
+const bucket = config.bucket;
+const emsProvider = config.ems_provider;
+const stackName = config.stackName;
+
+describe('The EMS report', () => {
+  describe('The EmsReport lambda is invoked daily', () => {
+    let expectReports = false;
+    beforeAll(async () => {
+      const lambda = new Lambda();
+      const lambdaConfig = await lambda.getFunctionConfiguration({ FunctionName: emsReportLambda })
+        .promise();
+      const lastUpdate = lambdaConfig.LastModified;
+
+      // Compare lambda function's lastUpate with the time 24 hours before now.
+      // If the lambda is created 24 hours ago, it must have been invoked
+      // and generated EMS reports for the previous day.
+      if (new Date(lastUpdate).getTime() < moment.utc().subtract(24, 'hours').toDate().getTime()) {
```

Comment:
This seems like something that would almost never be true, since new test executions come with a redeploy.  Am I missing something?

Reply:
If the function is not modified, kes is not going to deploy it again.